Formula/mmtf-cpp.rb

```diff
@@ -0,0 +1,40 @@
+class MmtfCpp < Formula
```

Comment:
If this is the correct citation, please add…
```
  # cite Bradley_2017: "https://doi.org/10.1371/journal.pcbi.1005575"
```
Try…
```sh
brew tap brewsci/base
brew cite --ruby https://doi.org/10.1371/journal.pcbi.1005575
brew cite https://doi.org/10.1371/journal.pcbi.1005575
brew cite --help
```
and once you've added it to the formula:
```sh
brew cite mmtf-cpp
```